x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -969,6 +966,71 @@ public void logRemovedField(String parserName, Supplier<XContentLocation> locati
         }
     }
 
+    private void doUpdateApiKey(
```

Comment:
One more iteration on this: I'm coming around to inlining 😅 

I've inlined the `executeRequest` method and in-lined building the request here. The only "functional" change is the noop change further down.